webpack.particle.js

```diff
@@ -103,7 +104,7 @@ module.exports = {
         test: /\.(png|jpg|gif|svg)$/,
         loader: 'file-loader',
         options: {
-          name: '[name].[ext]?[hash]',
+          name: 'images/[name].[ext]?[hash]',
```

Comment:
This change means that we need to update image twig paths. DO NOT MERGE until this is done.

Reply:
Solving this today.